antarest/study/storage/variantstudy/model/command/update_binding_constraint.py

```diff
@@ -52,6 +53,12 @@ def validate_series(
             return validate_matrix(v, values)
         return None
 
+    def _apply_config(self, study_data: FileStudyTreeConfig) -> CommandOutput:
+        bd_id = self.id
+        if bd_id not in study_data.bindings:
+            study_data.bindings.append(bd_id)
```

Comment:
the binding constraint should already exists, this should be a noop (nothing to do)